app/views/browse/nearby.html.erb

```diff
@@ -1,5 +1,5 @@
   <% if @spines.present? %>
-    <turbo-frame id="filmstrip_<%= params[:call_number].gsub(' ', '-') %>">
+    <turbo-frame id="filmstrip_<%= params[:call_number]&.gsub(' ', '-') %>">
```

Comment:
Should we even render this view if call_number isn't present? Can we add `params.require(:call_number)` into the controller?